src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedClientFeatures.java

```diff
@@ -22,7 +22,9 @@ public class UserDefinedClientFeatures extends LSPClientFeatures {
 
     public UserDefinedClientFeatures() {
         super();
-        // Add our completion feature
+
+        // Use the extended feature implementations
         setCompletionFeature(new UserDefinedCompletionFeature());
+        setWorkspaceSymbolFeature(new UserDefinedWorkspaceSymbolFeature());
```

Comment:
Using the extended workspace symbol feature just as we did the extended completion feature.